sections/futures/CrossMarginOnboard/CrossMarginFAQ.tsx

```diff
@@ -1,11 +1,13 @@
 import { useTranslation } from 'react-i18next';
 import styled from 'styled-components';
 
+export const CROSS_MARGIN_FAQ_URL = 'https://docs.kwenta.io/products/futures/cross-margin-accounts';
```

Comment:
We have `constants/links` but tend to put them in components when relevant. Do you think we should set a standard here?

Not to block this PR, but a good question: Where/how should we store/retrieve links?

Reply:
Good call, this should be in constants.